canvas_sdk/protocols/clinical_quality_measure.py

```diff
@@ -14,6 +14,7 @@ class Meta:
         description: str = ""
         information: str = ""
         references: list[str] = []
+        source_attributes: dict[str, str]
```

Comment:
@nmpsilva @kristenoneill  fyi, here is `source_attributes`, our new key-value store which is a property on the Meta class of ClinicalQualityMeasures